docs/extensions/tutorials/_posts/1970-01-14-SettingUpInstagram.md

```diff
@@ -0,0 +1,74 @@
+---
+layout: doc
+permalink: /docs/extensions/tutorials/setting-up-instagram
+title: Setting up Instagram
+section: Tutorials
+---
+
+# Setting up Instagram
+
+In order to show an Instagram feed in a Shoutem app using the Photos RSS, you will have to set up a redirect URL. The first step is to go to the Instagram developer [site] and create a new client if you don't already have one. Take note of the Client ID and Client Secret, you will be needing these. In `Valid redirect URIs` insert `https://new.shoutem.com`.
```

Comment:
You should link the `https://www.instagram.com/developer/clients/manage/` url because some people won't know where to go.